### DISCUSSION TITLE
writing.html

### DISCUSSION THREAD

**Discussion by wildhorse666** [jeessy2/ddns-go#1012] [Q&A]
大佬们打开web文件夹的writing.html浏览器提示报错，大佬们给看看
![8O)D`D0)J3AFVPIANXK0P12](https://github.com/jeessy2/ddns-go/assets/67013158/2df7af17-f6e8-464a-a067-32c32ace1b34)
![YN$@)8I{G9YJ`HRA)IKM` C](https://github.com/jeessy2/ddns-go/assets/67013158/e84ab1f9-e7e1-45a5-984f-32f9d2af9517)

**Comment by Mohsen2083** [comment]
![Screenshot 2024-02-13 120821](https://github.com/jeessy2/ddns-go/assets/131712492/0d30a5d2-bd64-4936-a22e-b50f7de5d142)

**Comment by yin1999** [comment]
`web/writing.html` 是一个模板文件，直接用浏览器打开是无效的，它需要通过模板引擎解析处理才能变成正确的 HTML 文件。
